python/dataAPIutilities.py

```diff
@@ -22,12 +22,7 @@ def getParameterList(functionObject):
         if parameterObject.name != "return":
             paramIndex += 1
             # get parameterObject.orig_declaration
-            declarationToUse = getOriginalDeclaration(parameterObject)
```

Comment:
Why don't I see the removal of the function definition getOriginalDeclaration in the diff?

Reply:
Oversight; it got forgotten when I changed from use_generate_mock_declaration.